api-lib/helpers/getUniqueAddress.ts

```diff
@@ -0,0 +1,44 @@
+import { HDNode } from '@ethersproject/hdnode';
+
+import { getAccountPath, SEED_PHRASE } from '../../scripts/util/eth';
+import { adminClient } from '../gql';
+
+export async function isAddressUnique(address: string): Promise<boolean> {
+  const { profiles } = await adminClient.query(
+    {
+      profiles: [
+        {
+          where: {
+            address: { _ilike: address },
+          },
+        },
+        {
+          id: true,
+        },
+      ],
+    },
+    {
+      operationName: 'isAddressUnique',
+    }
+  );
+
+  return profiles.length === 0;
+}
+
+export const getUniqueAddress = async () => {
+  let i = 100;
+
+  while (i > 0) {
+    const address = HDNode.fromMnemonic(SEED_PHRASE)
+      .derivePath(getAccountPath(Number(Math.random().toString().slice(2, 11))))
```

Comment:
I'm very surprised we're getting overlaps...

What if we just had a little helper function like:

function getRandomInt(max) {
  return Math.floor(Math.random() * max);
}

And then use that to choose a random number in the max getAccountPath range - which should REALLY not be already used, but in case it is then I guess we can verify it and try again.

Reply:
I got one conflict one time running the test, so I'm assuming if we have a bunch of tests running in parallel that that could happen. Thus, added the uniqueness check, we could potentially try and discover a safer way to generate unique addresses, one that comes to mind is for example hardcode a huge number of addresses and then just use them rather than generate them at run time. TBD

For now, applied your suggestion with https://github.com/coordinape/coordinape/pull/1486/commits/3d6cd6df32580e071a981f8b9f8d8a6c85889d1c